applications/swimming_DEM_application/custom_elements/spheric_swimming_particle.cpp

```diff
@@ -1417,7 +1417,14 @@ void SphericSwimmingParticle<TBaseElement>::Initialize(const ProcessInfo& r_proc
     mHasVirtualMassForceNodalVar = node.SolutionStepsDataHas(VIRTUAL_MASS_FORCE);
     mHasBassetForceNodalVar      = node.SolutionStepsDataHas(BASSET_FORCE);
     mHasLiftForceNodalVar        = node.SolutionStepsDataHas(LIFT_FORCE);
-    mSphericity                  = node.SolutionStepsDataHas(PARTICLE_SPHERICITY);
```

Comment:
Yes, it is more secure, although you are checking at every time step if the variable exists...